istio-telemetry/mixer-telemetry/templates/poddisruptionbudget.yaml

```diff
@@ -13,7 +13,7 @@ spec:
   minAvailable: 1
   selector:
     matchLabels:
-      app: istio-telemetry
+      app: telemetry
```

Comment:
mixer?

Reply:
Oof good catch, I was looking at the wrong place